MetaMask/metamask-extension
# [MV3] Add e2e chrome test job for MV3 build

## Explanation
This PR activates the e2e-chrome tests run on Circle Ci using the MV3 build.

These tests are not required to success in order to merge the PR to develop for now, as some are failing due to pending MV3 work (like the Backup functionality).

## More Information
Resolves https://github.com/MetaMask/metamask-extension/issues/16009 https://github.com/MetaMask/metamask-extension/issues/14326

## Screenshots/Screencaps

### Before

![image](https://user-images.githubusercontent.com/54408225/192512697-d96343a5-46e3-4787-ad93-2e458c008b54.png)

### After

![image](https://user-images.githubusercontent.com/54408225/192512952-9358de47-c68a-4511-9ed9-53817538ee21.png)


## Manual Testing Steps
1. Create a github branch
2. Add a commit
3. Push to Github
4. Check Circle Ci jobs and verify that `test-e2e-chrome-mv3` job is there

## Pre-Merge Checklist


### + If there are functional changes:
